net.go

```diff
@@ -1089,6 +1090,12 @@ func (m *Memberlist) decryptRemoteState(bufConn io.Reader, streamLabel string) (
 	moreBytes := binary.BigEndian.Uint32(cipherText.Bytes()[1:5])
 	if moreBytes > maxPushStateBytes {
 		return nil, fmt.Errorf("Remote node state is larger than limit (%d)", moreBytes)
+
+	}
+
+	//Start reporting the size before you cross the limit
+	if moreBytes > uint32(math.Floor(.06*maxPushStateBytes)) {
```

Comment:
Confirming 6% was intentional? Should it be a higher threshold like 80 or 90?

Reply:
Changed for debugging.  Intention was to have it at 60% to give time to upgrade fix things before you cross the mark.  If you do hit this bug chances are your cluster is large enough that upgrading is non-trivial.